requirements.txt

```diff
@@ -24,6 +24,7 @@ passlib>=1.6.2,<1.7
 lockfile>=0.10.2,<0.11
 python-gnupg>=0.3.7,<0.4
 jsonpath-rw>=1.3.0
+ldap
```

Comment:
Yep, that's fine for now (some more context is available here - https://github.com/StackStorm/st2/pull/1732#issuecomment-123224276).

Eventually, I need to spend a couple of days and make each auth backend a separate package so the user only installs the one they need.